app/benchmarks/benchmark_pfb_test.go

```diff
@@ -42,11 +42,7 @@ func BenchmarkCheckTx_PFB_Multi(b *testing.B) {
 		{blobSize: 400_000},
 		{blobSize: 500_000},
 		{blobSize: 1_000_000},
-		{blobSize: 2_000_000},
-		{blobSize: 3_000_000},
-		{blobSize: 4_000_000},
-		{blobSize: 5_000_000},
-		{blobSize: 6_000_000},
+		{blobSize: 2_000_000}, // maxTxSize is capped at 2MiB in checkTx
```

Comment:
these test cases were added if you want to manually bump the max tx size. I'm fine with removing them or keeping them